### PR TITLE
Allow two more customers to use the URL connectors. Requested by Paul…

### PR DESCRIPTION
**Pull request by lasryaric**
## Description

Allow two more customers to use the URL connectors. Requested by Pauline

### CODE REVIEW

**Review by fontanierh — APPROVED** [review]

**Comment by lasryaric** [comment]
@Yutcam is requesting us to hold on that before we get their approval. Not merging now.

**Pull request merged**
